libs/community/langchain_community/vectorstores/yellowbrick.py

```diff
@@ -24,19 +27,26 @@
 
 class Yellowbrick(VectorStore):
     """Yellowbrick as a vector database.
-
     Example:
         .. code-block:: python
             from langchain_community.vectorstores import Yellowbrick
             from langchain_community.embeddings.openai import OpenAIEmbeddings
             ...
     """
 
+    LSH_INDEX_TABLE: str = "_lsh_index"
+    LSH_HYPERPLANE_TABLE: str = "_lsh_hyperplane"
+    CONTENT_TABLE: str = "_content"
+
     def __init__(
         self,
         embedding: Embeddings,
         connection_string: str,
         table: str,
+        use_lsh: Optional[bool] = False,
```

Comment:
There are different ways to implement nearest neighbor searches.

even lsh can be parameterized in different ways. Is it possible to parameterize in a better way than a boolean?

Reply:
Great feedback. I've generalized the way of setting up indexes.